src/http-api/index.js

```diff
@@ -0,0 +1,37 @@
+var Hapi = require('hapi')
+var IPFS = require('../ipfs-core')
+var debug = require('debug')
+var log = debug('api')
+
+exports = module.exports
+
+exports.start = function (callback) {
+  // start IPFS and exports.ipfs = new IPFS()
+
+  exports.ipfs = new IPFS()
```

Comment:
Hmm, this looks like asking for trouble when the export is only available after `start` was called, also what will happen to module caching plus calling `start` twice?